<launch>

<node pkg="fwdis" name="fwdis_controller" type="fwdis_controller"/>

<node pkg="fwdis" name="fwdis_gazebo" type="fwdis_gazebo"/>

</launch>
